### PR TITLE
scala UDF image to construct a Image UDF from a URI

### LINKED ISSUE #645: `to_image` or `image` automatically registered Scala UDF
I will setup a Rikai production airflow dag, and I found it frustrating that we have to register a python UDF and then use it.

I want to setup the airflow dag using only rikai sql, I do not want to make a python whl (just to register the python UDF) for this production dag.

I think at least `to_image` should be re-impl in Scala, or we just use another udf name, like `image`.

### PR DESCRIPTION
**Pull request by da-tubi**
close #645

### CODE REVIEW

**Comment by changhiskhan** [comment]
test failure

File: build.sbt [hunk 1]
```diff
@@ -137,6 +137,22 @@ scalacOptions ++= {
 
 Test / parallelExecution := false
 Test / fork := true
+import Tests._
+{
+  def groupByFirst(tests: Seq[TestDefinition]) =
```
Comment (changhiskhan) [inline build.sbt:142]: Not sure I understand the problem. How did your change touch the Registry?

**Review by changhiskhan — COMMENTED** [review]
I'm concerned about hacking the testing like this.
Can we chat on slack when you're up?

File: build.sbt (hunk 1 above)
Comment (darcy-shen) [reply build.sbt:142]: Here is the official documentation of SBT: https://www.scala-sbt.org/1.x/docs/Testing.html#Forking+tests

**Review by changhiskhan — APPROVED** [review]

**Pull request merged**